wysadzulice/assets/js/home.js

```diff
@@ -0,0 +1,65 @@
+import jquery from 'jquery';
+import { Model, View, Collection} from 'backbone';
+
+const BannerView = View.extend({
+
+  initialize: function(options) {
+    this.parent = options.parent;
+    this.setElement(jquery('#home-banner'));
+    const markers = this.parent.collection.toApi();
+    const apiKey = 'https://maps.googleapis.com/maps/api/staticmap?size=400x300&key=AIzaSyCRnnZblqETt7ivbluhDXDuF3B0HrbdVnI' + markers;
+    this.$el.html(`<div id="banner-container"><img src="${apiKey}"></div>`);
+  },
+
+});
+
+const SlideView = View.extend({
+  initialize: function(options, slide) {
+    this.parent = options.parent;
+    this.setElement(jquery('#home-slider'));
+  },
+});
+
+
+const CampaignModel = Model.extend({
+  defaults: {
+    id: null,
+    lat: null,
+    lng: null,
+  },
+});
+
+const CampaignsCollection = Collection.extend({
+  model: CampaignModel,
+  url: '/api/campaigns/',
```

Comment:
I don't know if that's good idea to to define `url` here - by default `Backbone` is adding `id` at the end of `url` and I believe we will fetching it using something like `/api/campaigns/<lat>/<lng>/<lat>/<lng>`.